src/Data/TMap.hs

```diff
@@ -132,6 +134,24 @@ union :: TMap -> TMap -> TMap
 union = F.union
 {-# INLINE union #-}
 
+-- | The intersection of two 'TMap's using a combining function.
+--
+-- @O(n)@
```

Comment:
I just noticed this part in the documentation. I guess, It's more correct to say `O(n + m)` here. What do you think?

Reply:
I think it would be technically `O(min(n, m))` since it gives up when either of the maps are exhausted right?